utils/terrascript_client.py

```diff
@@ -2126,3 +2133,63 @@ def populate_tf_resource_elasticsearch(self, resource, namespace_info,
 
         for tf_resource in tf_resources:
             self.add_resource(account, tf_resource)
+
+    def populate_tf_resource_acm(self, resource, namespace_info,
+                                 existing_secrets):
+        account, identifier, common_values, \
+            output_prefix, output_resource_name = \
+            self.init_values(resource, namespace_info)
+
+        tf_resources = []
+        self.init_common_outputs(tf_resources, namespace_info,
+                                 output_prefix, output_resource_name)
+
+        secret = common_values.get('secret', None)
+        vault_data = vault_client.read_all(secret)
```

Comment:
i prefer to keep using secret_reader.
it abstracts away the fact that this is coming from Vault and allows us to provide this information through the configuration file if we want.